docs/markdown/Quick-guide.md

```diff
@@ -1,44 +1,84 @@
 ---
-title: Quick guide
-short-description: Guide to get started using meson
+title: Quickstart Guide
+short-description: Getting Started using mesonbuild
 ...
 
 # Using Meson
 
-Meson has been designed to be as easy to use as possible. This page
-outlines the basic use cases. For more advanced cases refer to Meson's
-command line help which is accessible with the command `meson --help`.
+Meson has been designed to be as simple to use as possible. This page
+outlines the initial steps needed for installation, troubleshooting,
+and standard use. 
 
-Requirements
---
+For more advanced configuration please refer to the command line help `meson --help` 
+or the meson documentation located at the [mesonbuild](https://mesonbuild.com) website.
+
+Table of Contents:
+* [Requirements](#requirements)
+* [Installation using package manager](#installation-using-package-manager)
+* [Installation using python](#installation-using-python)
+* [Installation from source](#installation-from-source)
+* [Troubleshooting](#troubleshooting)
+* [Compiling a meson project](#compiling-a-meson-project)
+* [Using meson as a distro packager](#using-meson-as-a-distro-packager)
 
-Meson has two main dependencies.
+Requirements:
```

Comment:
Adding colons on headings seems inconsistent.

Reply:
fixed in commit(5f28b40)